src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -253,6 +253,15 @@ private static boolean isDebugEnabled(BidRequest bidRequest, ExtBidRequest extBi
         return extRequestPrebid != null && Objects.equals(extRequestPrebid.getDebug(), 1);
     }
 
+    /**
+     * Returns events enabled flag for the given account.
+     * <p>
+     * This data is not critical, so returns false if null.
```

Comment:
Not relevant anymore.

Reply:
I can change to 
`Determines debug flag from {@link Account}` 
It would be ok ?

Reply:
Read once more - that's fine, never mind. Leave as is pls.